website/public/js/controllers/groupsCtrl.js

```diff
@@ -579,17 +578,24 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
         User.set({'invitations.party':{}});
       }
 
-      $scope.questCancel = function(){
+      $scope.questCancel = function(party){
         if (!confirm(window.env.t('sureCancel'))) return;
-        $rootScope.party.$questCancel();
+        Groups.questCancel(party);
       }
 
-      $scope.questAbort = function(){
+      $scope.questAbort = function(party){
         if (!confirm(window.env.t('sureAbort'))) return;
         if (!confirm(window.env.t('doubleSureAbort'))) return;
-        $rootScope.party.$questAbort();
+        Groups.questAbort(party);
       }
 
+      $scope.questAccept = function(party){
+        Groups.questAccept(party);
```

Comment:
Just tried it, and it seemed to work for me. 

Note, you'll have to change the buttons to call `ng-click='questAccept(party)'` and `ng-click='questReject(party)'` for it to work. 

We'll also probably want to add a test that the value gets updated.